packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -73,7 +73,7 @@ export class ClustersService extends ImmutableStore<ClustersServiceState> {
   }
 
   async restartClusterGatewaysAndCatchErrors(rootClusterUri: string) {
-    await Promise.all(
+    await Promise.allSettled(
```

Comment:
Don't we catch all errors from the `restartGateway`? 
I was sure that this would ensure that none of the promises would be rejected 🤔

Reply:
Hmm, true. But I suppose `allSettled` communicates the intent better? It says "I expect all of those promises to never reject and I want to wait for all of them", while `all` doesn't carry the same intention. So what do you say if we keep it that way?

Reply:
It was confusing for me because using `allSettled` and handling all errors for each promise (at the same time) suggested that there must be a case when the errors are not handled and we need `allSettled`. 

But it's a minor thing and I leave it to you to decide :)